getgather/browser/proxy.py

```diff
@@ -28,12 +33,23 @@ async def setup_proxy(
         None: If no proxy is configured
     """
     # Check if proxy service is configured
-    if not settings.BROWSER_HTTP_PROXY or not settings.BROWSER_HTTP_PROXY_PASSWORD:
-        logger.info(
-            "No proxy configured (BROWSER_HTTP_PROXY and BROWSER_HTTP_PROXY_PASSWORD not set)"
-        )
+    if not settings.BROWSER_PROXY:
         return None
 
+    proxy = urlsplit(settings.BROWSER_PROXY)
+    if not proxy.password:
+        raise ValueError("BROWSER_PROXY must contain a password")
```

Comment:
Can't password be optional? I imagine some internal proxy server (not for public) which does not require any password.